shopsdk/install.py

```diff
@@ -0,0 +1,322 @@
+	  l5 = f0 + f1 + f2 + f3 + f4 + f5 + f6 + f7 + f8 + f9
+	  l6 = g0 + g1 + g2 + g3 + g4 + g5 + g6 + g7 + g8 + g9
+	  l7 = h0 + h1 + h2 + h3 + h4 + h5 + h6 + h7 + h8 + h9
+	  l8 = i0 + i1 + i2 + i3 + i4 + i5 + i6 + i7 + i8 + i9
+	  l9 = j0 + j1 + j2 + j3 + j4 + j5 + j6 + j7 + j8 + j9
+	  zz = l0 + l1 + l2 + l3 + l4 + l5 + l6 + l7 + l8 + l9
+	  al=open(ao, y)
+	  for line in zz:
+	        al.write(str(line))
+	  al.close()
+	  av()
+	elif ar==9: #Mode 9 Identifier Software Check
+	  ap(w)
+	  ap("My Aquarium Custom Attachment Generator\n")
+	  ap("By John Pansera / Version 1.0\n")
+	  ap("Randomization Mod by 6100m, porting concepts by TMinusBlastedRocket, zero casting work of butch@stackoverflow\n")
+	  ab=aa.randint
+	  cb=ab(0, 2) #My Aquarium Tank Size Offset
+	  cd=ab(0, 5) #My Aquarium Glass Size Offset
+	  ce=ab(0, 4) #My Aquarium Floor Type Offset
+	  cf=ab(0, 4) #My Aquarium Background Type Offset
+	  cg=ab(0, 6) #My Aquarium Light Type Offset
+	  ch=ab(1, 12) #My Aquarium Special Date 1 Month Offset
+	  ci=ab(1, 31) #My Aquarium Special Date 1 Day Offset
+	  cj=ab(1, 12) #My Aquarium Special Date 2 Month Offset
+	  ck=ab(1, 31) #My Aquarium Special Date 2 Day Offset
+	  cl=ab(1, 12) #My Aquarium Special Date 3 Month Offset
+	  cm=ab(1, 31) #My Aquarium Special Date 3 Day Offset
+	  cn=ab(1, 101) #My Aquarium Add Fish Decisional Offset
+	  co=ab(1, 15) #My Aquarium Maximum Fish Offset
+	  ay=ag(cn) #Obsufcates variable to be shorter via aliasing
+	  az=ag(co) #Converts the My Aquarium Maximum Fish Offset to a ageger
+	  bb=list(range(1, 40))
+	  av=h.OrderedDict()
+	  av["unknown"]=j(0)  # Version?
+	  av["cb"]=j(ag(cb))
+	  av["cd"]=j(ag(cd))
+	  av["ce"]=j(ag(ce))
+	  av["cf"]=j(ag(cf))
+	  av["cg"]=j(ag(cg))
+	  av["cq"]=q(0)
+	  av["cr"]=l(0)
+	  av["cs"]=l(0)
+	  av["ct"]=j(0)
+	  av["cu"]=j(ag(ch))
+	  av["cv"]=j(ag(ci))
+	  av["ct_1"]=j(0)
+	  av["cw"]=j(0)
+	  av["cx"]=j(ag(cj))
+	  av["cy"]=j(ag(ck))
+	  av["cw_2"]=j(0)
+	  av["cz"]=j(0)
+	  av["da"]=j(ag(cl))
+	  av["dc"]=j(ag(cm))
+	  av["cz_1"]=j(0)
+	  data=ad.check(ay)
+	  if data=="even":
+	    if az <= 15:
+	      for i in range(az):
+	        sel=rn.choice(bb)
+	        if sel < 40:
+	          av["dd_%s" % i]=j(1)
+	          av["de_%s" % i]=j(sel)
+	          av["df_%s" % i]=q(0)
+	          av["dg_%s" % i]=q(0)
+	          av["dh_%s" % i]=q(1)
+	          av["di_%s" % i]=l(1)  # Birthday is day 1
+	          av["dj_%s" % i]=l(1)  # Current day set to 1
+	          av["dk"]=m(240 - (az * 16))
+	          av["dl"]=m(160)  # TODO: Add in object tables
+	          ap("Processing ...")
+	          f=e.BytesIO()
+	          for k, v in list(av.items()):
+                      f.write(ad.to_bytes(v))
+	          f.flush()
+	          f.seek(0)
+	          copy=f.read()
+	          crc32=format(g.crc32(copy) & 0xFFFFFFFF, '08x')
+	          f.close()
+	          ar=open('a0014682.dat', 'wb')  # Not sure how the name is generated yet
```

Comment:
I think it uses an ID based on your mail.

Reply:
You might be right actually, it does get transported via that right?